### PR TITLE
Race drinks!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Unironly mades like 12 new drinks for each race I can think of!
All the drinks give their race a +6 mood for 5 mins
If you drink the wrong race drink you get 25(!) disgust!
All drinks have a nice look imo and most are even animated.
Animates Mauna Loa
Changes 1 hats red cross to a blue!
Lowers cog champ fire rate
## Why It's Good For The Game

Adds more moodlits and better drinks for the barkeep to work for. Makes it so that not everyone loves the same drink!

## Changelog
:cl:
add: 12 new more drinks for most races!
add: New animations for mauna loa, and colour swap from red to blue for a Paramedic Hardsuit helm
tweak: Lowers cog champ ((the drink)) flare rate
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	color = "#EB1010" //(235, 16, 16)
+	description = "Strong mead mixed with more honey and ethanol. Known to beloved by most pallets."
+	boozepwr = 50 //strong!
+	taste_description = "honey and red wine"
+	glass_icon_state = "nord_king"
+	glass_name = "Keg of Nord King"
+	glass_desc = "A dripping keg of red mead."
+
+/datum/reagent/consumable/ethanol/nord_king/on_mob_life(mob/living/carbon/M)
+	if(ishumanbasic(M) || isdwarf(M) || isangel(M)) //Humans and angel races are rare
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2408]: >humans are rare

no they aren't?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+		quality = RACE_DRINK
+	else
+		M.adjust_disgust(25)
+		M.adjustToxLoss(1, 0) //Low tox do to being carp + jell toxins.
+	return ..()
+
+/datum/reagent/consumable/ethanol/laval_spit //Yes Laval
+	name = "Laval Spit"
+	color = "#DE3009" //(222, 48, 9)
+	description = "Heat minerals and some mauna loa. Meant for rock based live."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: life

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+
+/datum/reagent/consumable/ethanol/jell_wyrm
+	name = "Jell Wyrm"
+	color = "#FF6200" //(255, 98, 0)
+	description = "Horrable mix of Co2, toxins and heat. Meant for slime based live."
+	boozepwr = 40
+	taste_description = "tropical sea"
+	glass_icon_state = "jell_wyrm"
+	glass_name = "glass of Jell Wyrm"
+	glass_desc = "A bubbly drink that is rather inviting to those that don't know who is meant for."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: grammar check

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 4]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	if(ismammal(M) || iscatperson(M)) //well its not to bad for mammals
+		quality = RACE_DRINK
+	else
+		M.adjust_disgust(25)
+	return ..()
+
+/datum/reagent/consumable/ethanol/jell_wyrm
+	name = "Jell Wyrm"
+	color = "#FF6200" //(255, 98, 0)
+	description = "Horrable mix of Co2, toxins and heat. Meant for slime based live."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: horrible. Life, again.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 5]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	return ..()
+
+/datum/reagent/consumable/ethanol/hollow_bone
+	name = "Hollow Bone"
+	color = "#FCF7D4" //(252, 247, 212)
+	description = "Shockingly none-harmful mix of toxins and milk."
+	boozepwr = 15
+	taste_description = "Milk and salt"
+	glass_icon_state = "hollow_bone"
+	glass_name = "skull of Hollw Bone"
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: missed an o

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 6]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	glass_desc = "Mixing of milk and bone hurting juice for enjoyment for rather skinny people."
+
+/datum/reagent/consumable/ethanol/hollow_bone/on_mob_life(mob/living/carbon/M)
+	if(isplasmaman(M) || isskeleton(M))
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2510]: can plasma men actually drink this? I think the helmet blocks.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 7]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	if(ispodperson(M) || ismush(M))
+		quality = RACE_DRINK
+	else
+		M.adjust_disgust(25)
+	return ..()
+
+/datum/reagent/consumable/ethanol/darkbrew
+	name = "Darkbrew"
+	color = "#000000" //(0, 0, 0)
+	description = "Contained and acholic dark matter."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: spellcheck

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 8]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+
+/datum/reagent/consumable/ethanol/bug_zapper
+	name = "Bug Zapper"
+	color = "#F5882A" //(222, 250, 205)
+	description = "Metals and lemon juice. Hardly even a drink."
+	boozepwr = 5 //No booze really
+	taste_description = "copper and AC power"
+	glass_icon_state = "bug_zapper"
+	glass_name = "glass of Bug Zapper"
+	glass_desc = "An odd mix of copper, lemon juice and power meant for none-human consumption."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: non not none

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 9]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	if(isipcperson(M) || issynthliz(M))
+		quality = RACE_DRINK
+	else
+		M.adjust_disgust(25)
+	return ..()
+
+/datum/reagent/consumable/ethanol/nord_king
+	name = "Nord King"
+	color = "#EB1010" //(235, 16, 16)
+	description = "Strong mead mixed with more honey and ethanol. Known to beloved by most pallets."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: palettes

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 10]
```diff
@@ -2356,6 +2356,219 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	if(islizard(M))
+		quality = RACE_DRINK
+	else
+		M.adjust_disgust(25)
+	return ..()
+
+/datum/reagent/consumable/ethanol/oil_drum
+	name = "Oil Drum"
+	color = "#000000" //(0, 0, 0)
+	description = "Industeral grade oil mixed with some ethanol to make it a drink. Somehow not known to be toxic."
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2383]: spellcheck

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2510]: Take off hat and stuff they can. Also in 02-less area

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2408]: Kinda are tho

**Review by silicons — APPROVED** [review]

**Pull request merged**